apps/events/api/views.py

```diff
@@ -155,10 +156,18 @@ def unregister(self, request, pk=None):
         url_path="public-attendees",
     )
     def public_attendees(self, request, pk=None):
+        CACHE_TIME = 10
+        sentinel = object()
+        cached = cache.get("public-attendees" + str(pk), sentinel)
+        if cached is not sentinel:
+            return cached
+
```

Comment:
Much less code if you use `@method_decorator(cache_page(10))` https://www.django-rest-framework.org/api-guide/caching/#using-cache-with-apiview-and-viewsets

Reply:
Nice!